pkg/kudoctl/cmd/testdata/deploy-kudo-ns.yaml.golden

```diff
@@ -586,6 +586,9 @@ spec:
         - containerPort: 443
           name: webhook-server
           protocol: TCP
+        readinessProbe:
```

Comment:
Looks like it is readiness probe after all? The PR description needs updating, then.

Reply:
yeah, the startup probe is 1.16+

Reply:
Haven't we recently introduced something that requires k8s version >= 1.16 anyway?

Reply:
we did but I removed it again so we're 1.15 :)